app/javascript/lib/toasts.ts

```diff
@@ -8,16 +8,16 @@ const TYPE_TO_CLASS_MAP = {
   error: 'error',
 };
 
-export function toast(
-  message: string,
-  options?: { html?: boolean; type?: ToastType },
-) {
+type ToastifyOptions = {
+  type?: ToastType;
+};
+
+export function toast(message: string, options?: ToastifyOptions) {
   Toastify({
     text: message,
     position: 'center',
     duration: 3000,
     className: options?.type && TYPE_TO_CLASS_MAP[options?.type],
-    escapeMarkup: !options?.html,
```

Comment:
I'm removing the option to render HTML toasts via toastify, because all current such use cases are now being handled via `vueToast` rather than via this `toast` method, and, if possible, `vueToast` should probably be generally preferred for such needs.